index.html

```diff
@@ -10711,7 +10711,47 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				</tbody>
 			</table>
 		</div>
-                <div class="property" id="aria-details">
+		<div class="property" id="aria-description">
+			<pdef>aria-description</pdef>
+			<div class="property-description">
+				<p>Defines a string value that describes or annotates the current element. See related <pref>aria-describedby</pref>.</p>
+				<p>The <pref>aria-description</pref> attribute is similar to the <pref>aria-label</pref> in that both profide a flat string to associate with the element, but a label should be concise, where a description is intended to provide more verbose information.</p>
+				<p>The purpose of <pref>aria-description</pref> is the same as that of <pref>aria-describedby</pref>. It provides the user with additional descriptive text for the object. The most common <a>accessibility <abbr title="Application Programing Interfaces">API</abbr></a> mapping for a description is the <a>accessible description</a> property.</p>
+				<p>If the description text is visible on screen, authors SHOULD NOT use <pref>aria-description</pref>.</p>
+				<ul>
+					<li>Authors SHOULD use <pref>aria-details</pref> when the description or annotation is large, complex or has useful semantics or structure. Using <pref>aria-details</pref> will allow assistive technology users to visit the structured content and provide additional navigation commands within the content.</li>
+					<li>Authors SHOULD use <pref>aria-describedby</pref> when the desription or annotation is very small and simple, and is best experienced as a flat string.</li>
```

Comment:
Should this be `aria-description` instead of `aria-describedby`? Or should the information also include that the information is visible? Otherwise it doesn't seem clear enough why you would use `aria-description`.

Reply:
Hi, this bulleted list is trying to say what to use if the description is visible on screen. In that case, it doesn't make to use aria-description. They should use aria-details or aria-describedy in that case. I've changed the paragraph before to try to make it clearer that the list below is what to use instead, when the text is available in the page.

Reply:
Ah gotcha - update is a lot clearer now, thanks!